awswrapper/s3.go

```diff
@@ -292,16 +298,18 @@ func (o *S3Service) RemoveAllFromS3(bucketName string, path string) (err error)
 	return
 }
 
-// UploadToS3Concurrently uploads content to S3 concurrently
-// args. [0]. Make Public Or Not.
-func (o *S3Service) UploadToS3Concurrently(content []byte, bucketName string, path string, args ...interface{}) error {
-
-	session := session.Must(session.NewSession(&aws.Config{
-		Region: aws.String(o.region),
-	}))
+func (o *S3Service) UploadToS3Concurrently(reader io.Reader, bucket, path string, contentType *string, options *S3Options, args ...interface{}) (err error) {
```

Comment:
Can we also remove the `contentType *string` in `UploadToS3Concurrently`? Because the `UploadToS3Concurrently` is an exposed method and now we are leaving it up to the user to determine it instead of figuring it out ourselves.

Reply:
We can remove `contentType *string` and follow this way to determine the file type from `io.Reader`.
https://dev.to/knqyf263/determine-a-file-type-of-io-reader-2hp4

What do you think? Does this seem like more work?

Reply:
We can leave it the same. Is this used in Inspect API service? As far as I know, only the converter uses it.